students/headieh_godwin/lesson09/donor_models.py

```diff
@@ -70,47 +68,45 @@ def list_donors(self):
         return self.donor_obs_list
 
     @list_donors.setter
-    def list_donors(self, donors = []):
+    def list_donors(self, donors):
+        """list of donors"""
         self.donor_obs_list.extend(donors)
 
+    def find_donor(self, new_donor):
+        """find donor in list"""
+        return next((donor for donor in self.donor_obs_list
+                     if(donor._name.lower() == new_donor.lower())
+                     ), None)
```

Comment:
you could have used a dictionary to store your donor object for easier look ups - where a key is donor name and value is donor object